share/spack/setup-env.fish

```diff
@@ -0,0 +1,826 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+#################################################################################
+#
+# This file is part of Spack and sets up the spack environment for the friendly
+# interactive shell (fish). This includes dotkit support, module support, and it
+# also puts spack in your path. The script also checks that at least module
+# support exists, and provides suggestions if it doesn't. Source it like this:
+#
+#    source /path/to/spack/share/spack/setup-env.fish
+#
+#################################################################################
+# This is a wrapper around the spack command that forwards calls to 'spack use'
+# and 'spack unuse' to shell functions. This in turn allows them to be used to
+# invoke dotkit functions.
```

Comment:
I think dotkit has been removed now.

Reply:
Done! I've fixed the comments to match the `setup-env.sh`.